packages/react/menu/src/Menu.tsx

```diff
@@ -604,7 +604,7 @@ const MenuContentImpl = React.forwardRef<MenuContentImplElement, MenuContentImpl
                     {...popperScope}
                     {...contentProps}
                     ref={composedRefs}
-                    style={{ outline: 'none', ...contentProps.style }}
+                    style={{ ...contentProps.style }}
```

Comment:
This can be removed as props are spread anyway right above.

Reply:
i think benoit is proposing this change:

```diff
-                     style={{ ...contentProps.style }}
```